test/ox-jira-test.el

```diff
@@ -144,7 +146,14 @@ h1. {color:red}{{TODO}}{color} This is another headline {color:blue}{{:FOO:BAR:}
   (should (equal "see [This Thing|#This Thing] for details\n"
                  (to-jira "see [[#Heading Name][This Thing]] for details")))
   (should (equal "contact [User Name|~accountid:accountid] for details\n"
-                 (to-jira "contact [[~accountid:accountid][User Name]] for details\n"))))
+                 (to-jira "contact [[~accountid:accountid][User Name]] for details\n")))
+  (cl-letf (((symbol-function #'org-export-custom-protocol-maybe)
+             (lambda (&rest _) "http://example.com")))
+    (should (equal "fi [http://example.com] fo\n"
+                   (to-jira "fi [[example-protocol://example.com]] fo")))
+    (should (equal "fi [Example|http://example.com] fo\n"
+                   (to-jira "fi [[example-protocol://example.com][Example]] fo"))))
+  )
```

Comment:
Can you pull up this `)` to the previous line please?

Reply:
Done